auth0/src/test/java/com/auth0/android/provider/RedirectActivityTest.java

```diff
@@ -49,13 +52,13 @@ public void shouldLaunchAuthenticationActivityWithDataOnSuccess() {
         createActivity(resultIntent);
         activityController.create().start().resume();
 
-        Intent authenticationIntent = shadowOf(activity).getNextStartedActivity();
```

Comment:
Use cases tested using a shadow that keeps only what we need to assert: that it started a specific activity and that it was then finished.